src/features/layer_features_panel/atoms/helpers/openaerialmap.ts

```
@@ -0,0 +1,123 @@
+    return 'unknown';
+  }
+  if (sizeBytes > 1000000000) {
+    return `${parseFloat((sizeBytes / 1000000000).toFixed(2))} GB`;
+  }
+  if (sizeBytes > 1000000) {
+    return `${parseFloat((sizeBytes / 1000000).toFixed(2))} MB`;
+  }
+  if (sizeBytes > 1000) {
+    return `${parseFloat((sizeBytes / 1000).toFixed(2))} KB`;
+  }
+  return `${parseFloat((sizeBytes / 1000).toFixed(2))} B`;
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)